entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+### Usage: /entrypoint.sh <command> <args>
+set -e
+command=$1
+
+if [ "$command" == "serve" ]; then
+  file="$2"
+  gomplate -f "$file" -o "/etc/dex/config.yaml";
```

Comment:
This is a bit problematic. Ideally, we want users to use this mount point going forward (eg. we could add some sort of config discovery, so manually choosing a config file becomes optional and `/etc/dex/config.yaml` is loaded automatically). This command will simply overwrite that file all the time.

In addition, they might also mount `/etc/dex` (eg. to mount certs), so it would actually become readonly, meaning we can't write the rendered template there.

`/tmp` sounds like a potential destination, `/var/run`/`/run` as well, but I'm not sure how we should tackle this problem yet.

Reply:
/tmp sounds good for me